finality-aleph/src/party.rs

```diff
@@ -258,20 +262,27 @@ where
             .runtime_api()
             .current_session(&BlockId::Number(prev_block_number))
         {
-            Ok(session) => session,
+            Ok(session) => {
+                self.sessions.insert(
+                    session_id,
+                    SessionInstance {
+                        session: session.clone(),
+                    },
```

Comment:
I was wondering if we could replace the value type of the hashmap to `Session<_, _>` and get rid of the `SessionInstance` type completely?